daemon/graphdriver/lcow/lcow_svm.go

```diff
@@ -296,7 +297,9 @@ func (svm *serviceVM) createUnionMount(mountName string, mvds ...hcsshim.MappedV
 	}
 
 	logrus.Debugf("Doing the overlay mount with union directory=%s", mountName)
-	if err = svm.runProcess(fmt.Sprintf("mkdir -p %s", mountName), nil, nil, nil); err != nil {
+	errOut := &bytes.Buffer{}
+	if err = svm.runProcess(fmt.Sprintf("mkdir -p %s", mountName), nil, nil, errOut); err != nil {
+		logrus.Errorf("mkdir -p %s failed: %s (%s)", mountName, err, errOut.String())
```

Comment:
would wrapping and returning the error work?

```go
return errors.Wrapf(err, "mkdir -p %s failed (%s)", mountName, errOut.String())
```

Reply:
Oh; this came up recently; when logging, we should prefer `logrus.WithError()`

idk, something like this (but may need some tweaking)

```go
logrus.WithError(err).WithField("mountName", mountName).Errorf("mkdir failed (%s)", errOut)
```

Reply:
Actually, went back to wrapping the error, and updated the call-site so that it doesn't actually throw it away on Windows (where there's no back-compat on error messages needed), and verified. Let's see if that message change trips up a CI test.